script/spm_bootstrap

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+SWIFT_SNAPSHOT=$1
+
+cp -r /Library/Developer/Toolchains/$SWIFT_SNAPSHOT.xctoolchain/usr/local/include/clang-c /usr/local/include/clang-c
+cp Source/Clang_C/module.modulemap /usr/local/include/clang-c/module.modulemap
```

Comment:
For using from `curl …| sh`, `module.modulemap` also needed to create here or download from.